userguide/system.rst

```diff
@@ -1793,9 +1793,11 @@ appears so the current configuration can be saved to external media.
 After clicking :guilabel:`INSTALL MANUAL UPDATE FILE`, choose a
 location to temporarily store the update file on the %brand% system.
 Use :guilabel:`Browse` to navigate to the downloaded manual update
-file (:file:`-manual-update-unsigned.tar`). Finally, click
-:guilabel:`APPLY UPDATE` to begin the update. A progress dialog is
-displayed during the update. **Do not** interrupt the update.
+file (:file:`-manual-update-unsigned.tar`). Set
```

Comment:
```suggestion
file ending with :file:`-manual-update-unsigned.tar`. Set
```

Reply:
Trying to avoid parentheses.

Reply:
This really needs to be two sentences, because it's trying to say two things:
1. Manual update file names end with "-manual-update-unsigned.tar".
2. Use Browse to locate the downloaded manual update.